recipes/default.rb

```diff
@@ -51,3 +51,10 @@
   user node['timesync']['user']
   action :start_or_graceful_reload
 end
+
+directory '/opt/timesync/source' do
```

Comment:
I confirm that it is owned by room without creation of this directory. @ramereth Leave it as it? Or I could use a chown -R?